pulsar-broker/src/main/java/org/apache/pulsar/broker/lookup/TopicLookupBase.java

```diff
@@ -126,14 +128,17 @@ protected void internalLookupTopicAsync(TopicName topicName, boolean authoritati
 
     private void validateAdminAndClientPermission(TopicName topic) throws RestException, Exception {
         try {
-            validateAdminAccessForTenant(topic.getTenant());
+            validateTopicOperation(topic, TopicOperation.LOOKUP);
         } catch (Exception e) {
-            checkConnect(topic);
+            // unknown error marked as internal server error
+            log.warn("Unexpected error while authorizing TopicOperation.LOOKUP. topic={}, role={}. Error: {}",
```

Comment:
Do we need to log here? The exception would be expected if the client is not authorized and since it's already bubbled up, we would end up logging that twice.

Reply:
Dropped in 596d85f418efe40b50114da3d4a6d126cb796882.

PR rebased due to a recent conflict.